### PR TITLE
restructure membership and sponsorship info

### PR DESCRIPTION
**Pull request by tracykteal**
Related to #97 and #98 

- Created Sponsor page under About that has info on Sponsors and links to Member Organisations
- Updated Membership Organisation page with a list rather than the logos, since those were outdated

### CODE REVIEW

File: pages/sponsors.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+permalink: "/sponsors/"
+---
+
+The Carpentries is a fiscally sponsored project of the 501(c)3 [Community Initiatives](http://communityin.org/)
+
+
+### Our Member Organisations
+
+A Member Organisation is an organisation that has made a committment to
+the growth and spread of The Carpentries and building capacity for training
```
Comment (kariljordan) [inline pages/sponsors.md]: what about a financial commitment?

Comment (kariljordan) [reply pages/sponsors.md]: A Member Organization is an organization that has made a financial commitment to the growth and spread of The Carpentries through building capacity for training locally.

**Comment by elizabethwilliams8** [comment]
Looks good to me! Will the list be permanent, or will I gather logos add them back in when we have them?

File: pages/sponsors.md (hunk 1 above)
Comment (tracykteal) [reply pages/sponsors.md]: thanks, I'll incorporate that edit

**Comment by tracykteal** [comment]
I think it would be better to have the logos. So, even though we put this up for now, we should start drafting a membership page that will have the logos. Maybe something like the layout of the Teams page.

**Comment by tracykteal** [comment]
Thanks! With two LGTMs, I'll merge.

**Pull request merged**